### DISCUSSION TITLE
How can I add bootstrap classes to tooltip components?

### DISCUSSION THREAD

**Discussion by kabirpathak** [gilbarbara/react-joyride#721] [Q&A] [answered]
In my project, I've implemented a tour. However, the styling does not match the overall styling of the website. My aim is to use the same bootstrap classes in joyride that i have used in my website.
Example. 'btn bg-primary btn-sm btn btn-secondary' for the next and back buttons. How can I do that?

**Comment by gilbarbara** [comment, marked as answer]
hey @kabirpathak 

You can use [custom components](https://docs.react-joyride.com/custom-components).
